staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -368,9 +367,7 @@ func (wc *watchChan) sendError(err error) {
 
 func (wc *watchChan) sendEvent(e *event) {
 	if len(wc.incomingEventChan) == incomingBufSize {
-		klog.V(3).Infof("Fast watcher, slow processing. Number of buffered events: %d."+
-			"Probably caused by slow decoding, user not receiving fast, or other processing logic",
-			incomingBufSize)
+		klog.V(3).InfoS("Fast watcher, slow processing. Probably caused by slow decoding, user not receiving fast, or other processing logic", "Incoming bufferedEvents", incomingBufSize)
```

Comment:
```suggestion
		klog.V(3).InfoS("Fast watcher, slow processing. Probably caused by slow decoding, user not receiving fast, or other processing logic", "incomingEvents", incomingBufSize)
```

Reply:
Please use [lowerCamelCase](https://en.wiktionary.org/wiki/lowerCamelCase)  as described in migration instructions.